testsuite/python/tests_common.py

```diff
@@ -594,3 +594,7 @@ def single_component_maxwell(x1, x2, kT):
     """Integrate the probability density from x1 to x2 using the trapezoidal rule"""
     x = np.linspace(x1, x2, 1000)
     return np.trapz(np.exp(-x**2 / (2. * kT)), x) / np.sqrt(2. * np.pi * kT)
+
+
+def lists_contain_same_elements(list1, list2):
```

Comment:
Isn't that just `set(list1) == set(list2)` with extra steps?

Reply:
no, because then the order does matter which is not what the function name advertises

Reply:
```python
assert set([1,2,3]) == set([3,2,1])
```

Reply:
this requires the elements to be hashable...